Crossover patcher/en.lproj/Localizable.strings

```diff
@@ -1,95 +1,126 @@
-"AgreeAndProceedButtonText" = "Agree and Proceed";
-"AllowRepatchUpgradeToggle" = "Allow repatch / upgrade";
+/* --- App Info & Credits --- */
+"CXPatcherName" = "CXPatcher";
+"CrossOverPatcher" = "CXPatcher";
 "Credits" = "Credits:";
-"CreditsLinks" = "[@gcenx](https://github.com/Gcenx)
-[@nastys](https://github.com/nastys)";
-"CreditsText" = "Many thanks to the developers behind the DXVK and MoltenVK patches for their great help and for providing the binaries.";
+"CreditsText" = "Huge thanks to the developers behind DXVK and MoltenVK patches for their great help and providing the binaries.";
+"CreditsLinks" = "[@gcenx](https://github.com/Gcenx)\n[@nastys](https://github.com/nastys)";
```

Comment:
I'll find a way to avoid putting the url in the locale files at some point, but for now i guess we'll leave it like this